src/com/lftechnology/java/training/alina/io/findinteger/IntReader.java

```diff
@@ -0,0 +1,34 @@
+package com.lftechnology.java.training.alina.io.findinteger;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A file which has some int value among with other string value . Write a program that gets the int piece of data. What is the int data?
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class IntReader {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)